scripts/cluster

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Reset test-dirs
+rm -rf test-chain-*
+rm -f genesis.json
+
+go build -o polygon-edge .
+
+nodeid=$(./polygon-edge secrets init --data-dir test-chain- --num 4 | grep Node | head -n 1 | awk -F ' ' '{print $4}')
+
+./polygon-edge genesis --consensus ibft --ibft-validators-prefix-path test-chain- \
+  --bootnode /ip4/127.0.0.1/tcp/10001/p2p/"$nodeid" \
```

Comment:
Didn't we have a PR to use all nodes as bootnodes?

Reply:
The functionality is only on `v3-parity` I think. If `--bootnode` is not specified, all the validators are set as bootnodes

Reply:
Not sure about this